cmd/bundle/init.go

```diff
@@ -15,6 +15,26 @@ var gitUrlPrefixes = []string{
 	"git@",
 }
 
+// TODO: add tests for these functions
+func isRepoUrl(url string) bool {
+	result := false
+	for _, prefix := range gitUrlPrefixes {
+		if strings.HasPrefix(url, prefix) {
+			result = true
+			break
+		}
+	}
+	return result
+}
+
+// Computes the repo name from the repo URL. Treats the last non empty word
+// when splitting at '/' as the repo name. For example: for url git@github.com:databricks/cli.git
+// the name would be "cli.git"
+func repoName(url string) string {
```

Comment:
won't this panic if I just pass incorrect input like `git@github.com:databricks` ?

Reply:
It passes the input as is. Because the slice returned by the sep function is never empty